src/renderers/ascii_default/mod.rs

```diff
@@ -0,0 +1,226 @@
+mod marks;
+mod styles;
+
+#[cfg(feature = "ansi_term")]
+use crate::renderers::ascii_default::styles::color::Style;
+#[cfg(feature = "termcolor")]
+use crate::renderers::ascii_default::styles::color2::Style;
+#[cfg(all(not(feature = "ansi_term"), not(feature = "termcolor")))]
+use crate::renderers::ascii_default::styles::plain::Style;
```

Comment:
This global compile time feature selection is a misuse of cargo features. If I turn on both the `ansi_term` and `termcolor` features, this will fail to compile.

Features should be purely additive. Turning on features gives more options, not changes the behavior of code that works both with and without the feature.

Reply:
I agree those should not be mutually exclusive. Similarly to renderer choices, I'll move that to be a selector with options enabled via features.